packages/react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -27,6 +29,8 @@ import { usePrefix } from '../../internal/usePrefix';
 import { useSavedCallback } from '../../internal/useSavedCallback';
 import { FormContext } from '../FluidForm';
 import { WarningFilled, WarningAltFilled } from '@carbon/icons-react';
+import { ReactAttr } from '../../types/common';
+//import {FlatpickrFn} from "flatpickr/dist/types/instance";
```

Comment:
Can we delete this?

Reply:
Thanks @tw15egan . I've removed the line.